packages/odyssey-react-mui/src/labs/NavAccordion.tsx

```diff
@@ -0,0 +1,162 @@
+   */
+  isDefaultExpanded?: boolean;
+  /**
+   * Whether the item is disabled
+   */
+  isDisabled?: boolean;
+  /**
+   * Whether the item is expanded
+   */
+  isExpanded?: boolean;
+  /**
+   * Event fired when the expansion state of the accordion is changed
+   */
+  onChange?: MuiAccordionProps["onChange"];
+  /**
+   * The icon element to display at the start of the Nav Item
+   */
+  startIcon?: ReactNode;
+} & (
+  | {
+      isExpanded: boolean;
+      isDefaultExpanded?: never;
+    }
+  | {
+      isDefaultExpanded?: boolean;
+      isExpanded?: never;
+    }
+) &
+  Pick<HtmlProps, "testId" | "translate">;
+
+const AccordionLabelContainer = styled("span", {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" && prop !== "isIconVisible",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  isIconVisible: boolean;
+}>(({ odysseyDesignTokens, isIconVisible }) => ({
+  width: "100%",
+  marginLeft: isIconVisible ? odysseyDesignTokens.Spacing3 : 0,
+  fontSize: odysseyDesignTokens.TypographyScale0,
+  fontWeight: odysseyDesignTokens.TypographyWeightHeading,
+  color: odysseyDesignTokens.TypographyColorHeading,
+  alignSelf: "center",
+}));
+
+const NavAccordion = ({
+  children,
+  label,
+  id: idOverride,
+  isDefaultExpanded,
+  isDisabled,
+  isExpanded,
+  translate,
+  startIcon,
+}: NavAccordionProps) => {
+  const id = useUniqueId(idOverride);
+  const headerId = `${id}-header`;
+  const contentId = `${id}-content`;
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+
+  return (
+    <MuiAccordion
+      defaultExpanded={isDefaultExpanded}
+      disabled={isDisabled}
+      disableGutters
+      expanded={isExpanded}
+      sx={{
+        border: "0 !important",
+        width: "100%",
+      }}
+    >
+      <MuiAccordionSummary
+        sx={{
+          padding: `${odysseyDesignTokens.Spacing2} ${odysseyDesignTokens.Spacing4}`,
+          ".MuiAccordionSummary-expandIconWrapper.Mui-expanded": {
+            transform: "rotate(-90deg) !important",
+          },
+        }}
+        aria-controls={contentId}
+        expandIcon={<ChevronRightIcon />}
+        id={headerId}
+      >
+        <Support component="div" translate={translate}>
+          <Box
+            sx={{
+              display: "flex",
+              alignItems: "center",
+            }}
+          >
+            {startIcon && startIcon}
+            <AccordionLabelContainer
+              odysseyDesignTokens={odysseyDesignTokens}
+              isIconVisible={!!startIcon}
+            >
+              {label}
+            </AccordionLabelContainer>
+          </Box>
+        </Support>
+      </MuiAccordionSummary>
+      <MuiAccordionDetails
+        aria-labelledby={headerId}
+        sx={{
+          paddingTop: 0,
+          paddingBottom: 0,
+          paddingLeft: odysseyDesignTokens.Spacing2,
+        }}
+      >
+        {children}
+      </MuiAccordionDetails>
+    </MuiAccordion>
+  );
+};
```

Comment:
This approach is working for now, but it feels like we're fighting against the `MuiAccordion` style customizations in `components.tsx`.

There are two approaches that would be more sustainable:
1. Pull out the existing accordion customizations into `Accordion.tsx`, so that each accordion's styles are bounded to that specific accordion
2. Add some kind of namespacing by adding a CSS class to the root of each accordion, and then targeting that class in `components.tsx`

Reply:
done